Item23_Understand_std_move_and_std_forward/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14 -stdlib=libc++")
+set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14")
```

Comment:
If I remember correctly, I had to add -stdlib=libc++ to get things to compile with clang.  Will this still compile with clang if you remove this option?